src/microphone/sampler.ts

```diff
@@ -40,8 +52,20 @@ export class Downsampler implements AudioFilter {
   }
 }
 
-export function generateDownsampler(sourceSampleRate: number, targetSampleRate: number): Downsampler {
-  return new Downsampler(sourceSampleRate, targetSampleRate, generateFilter(sourceSampleRate, targetSampleRate / 2, 23))
+export function newSampler(sourceSampleRate: number, targetSampleRate: number): AudioFilter {
+  if (sourceSampleRate === targetSampleRate) {
+    console.log('bypass')
```

Comment:
The `console.log`s here and below are probably not needed.

Reply:
Good catch, removed! 👍